internal/olm/client.go

```diff
@@ -60,11 +56,19 @@ func ClientForConfig(cfg *rest.Config) (*Client, error) {
 		Client:          cl,
 		HTTPClient:      *http.DefaultClient,
 		BaseDownloadURL: "https://github.com/operator-framework/operator-lifecycle-manager/releases",
+		Namespace:       DefaultNamespace,
 	}
 	return c, nil
 }
 
 func (c Client) InstallVersion(ctx context.Context, version string) (*olmresourceclient.Status, error) {
+
+	var (
```

Comment:
could it be const or we are changing these values?

Reply:
I will change this const

Reply:
Instead of setting these here, make their names constants at the top and create each key when needed:
```go
const (
	olmOperatorName     = "olm-operator"
	catalogOperatorName = "catalog-operator"
	packageServerName   = "packageserver"
)


func (c Client) InstallVersion(ctx context.Context, namespace, version string) (*olmresourceclient.Status, error) {
	...
	
	log.Print("Waiting for deployment/olm-operator rollout to complete")
	olmOperatorKey := types.NamespacedName{Namespace: namespace, Name: olmOperatorName}
	if err := c.DoRolloutWait(ctx, olmOperatorKey); err != nil {
	...
}
```

Reply:
fixed